comps/embeddings/predictionguard/requirements.txt

```diff
@@ -0,0 +1,9 @@
+docarray
+fastapi
+predictionguard
```

Comment:
Should we pin the specific versions of these dependencies to ensure that the build is stable?